src/handwritten/types.py

```diff
@@ -54,3 +54,9 @@
 
 # endregion
 
+# region ID Pin namedtuples
+
+IDPinMemory = collections.namedtuple(
```

Comment:
This cluster is not the memory.

IDPinData would be more accurate, but that's confusing because one of the fields is called `data`. (The other field, `format_code`, is also data, but not that kind of data.)

Maybe IDPinContents? 

I honestly don't think using a simple tuple was so bad.

@zhindes